libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/AudioCapabilities.java

```diff
@@ -102,25 +100,33 @@ public static AudioCapabilities getCapabilities(Context context) {
     if (Util.SDK_INT >= 23 && Api23.isBluetoothConnected(context)) {
       return DEFAULT_AUDIO_CAPABILITIES;
     }
+    ImmutableSet.Builder supportedEncodings = new ImmutableSet.Builder<>();
+
```

Comment:
nit: Remove the empty line 104 or swap the line 103 and 104, to make the construction of the ImmutableSet.Builder and its usage closer.

Reply:
@tianyif Done. 51b2201f3ae79c09f5b60b11a6bdf4361e00481a